_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -16,15 +16,16 @@ input_aspects:
 acknowledgements:
   authors:
     - Anne Thyme Nørregaard
+    - Jean-Yves Moyen
 ---
 
 ## Applicability
 
-Any [WAI-ARIA state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is specified on an HTML or SVG element that is [included in the accessibility tree][].
+Any [WAI-ARIA state or property][] that is specified on an HTML or SVG element that is [included in the accessibility tree][].
```

Comment:
There are currently no SVG examples. Should we add some?

Reply:
Perhaps one for good measure. Is there an aria state or property that is more often used with SVG rather than HTML?

Reply:
It looks like the roles defined in the WAI-ARIA Graphic module only accept the global states and properties… I've added an example anyway.